### PR TITLE
Feat(YouTube & Music - Branding): Changed Default Options

### PR DESCRIPTION
**Pull request by Xisrr1**
**YouTube and YT music:**

`Revancify Yellow` is the default selected icon.

### CODE REVIEW

**Comment by anddea** [comment]
@Francesco146, what do you think, should we enable `Custom branding icon` by default? I excluded it initially to stay closer to the original YouTube client, which applies to most other default options as well.

**Comment by Francesco146** [comment]
@anddea i'm of the same opinion as you, it's better to keep consistency with everything else. also since, from my point of view, when i patch any application, i don't expect the icon to be changed without my explicit choice. advice for the future is to create an issue or discussion, before taking the initiative and doing a PR

**Comment by anddea** [comment]
Revancify Yellow can be set as default though. But the patch should not be included by default.

File: src/main/kotlin/app/revanced/patches/youtube/layout/branding/icon/CustomBrandingIconPatch.kt [hunk 1]
```diff
@@ -20,9 +20,9 @@ object CustomBrandingIconPatch : BaseResourcePatch(
     description = "Changes the YouTube app icon to the icon specified in options.json.",
     dependencies = setOf(SettingsPatch::class),
     compatiblePackages = COMPATIBLE_PACKAGE,
-    use = false,
+    use = true,
```
Comment (Francesco146) [inline src/main/kotlin/app/revanced/patches/youtube/layout/branding/icon/CustomBrandingIconPatch.kt:23]: `use = false`

File: src/main/kotlin/app/revanced/patches/music/layout/branding/icon/CustomBrandingIconPatch.kt [hunk 2]
```diff
@@ -19,9 +19,9 @@ object CustomBrandingIconPatch : BaseResourcePatch(
     name = "Custom branding icon for YouTube Music",
     description = "Changes the YouTube Music app icon to the icon specified in options.json.",
     compatiblePackages = COMPATIBLE_PACKAGE,
-    use = false,
+    use = true,
```
Comment (Francesco146) [inline src/main/kotlin/app/revanced/patches/music/layout/branding/icon/CustomBrandingIconPatch.kt:22]: `use = false`

**Review by Francesco146 — CHANGES_REQUESTED** [review]

**Comment by zerozero0022** [comment]
I'm sorry I'm disturbing you.

I think it's okay to set anything as the default because the user can change the icon.

**Comment by Xisrr1** [comment]
> I'm sorry I'm disturbing you.
> 
> I think it's okay to set anything as the default because the user can change the icon.

@anddea and @Francesco146 are deciding, maybe it'll change in the future.

**Review by Francesco146 — APPROVED** [review]

**Comment by Francesco146** [comment]
thanks

**Pull request merged**